### ISSUE TITLE
Compiler errors in current release

### ISSUE THREAD

**Issue by mmstick** [rustic-games/jilu#11]
This seems to be an issue now, even for installing from Crates.io

```
error[E0308]: mismatched types
  --> src/changelog.rs:70:47
   |
70 |         let mut log = tera.render("template", context)?;
   |                                               ^^^^^^^
   |                                               |
   |                                               expected reference, found struct `tera::context::Context`
   |                                               help: consider borrowing here: `&context`
   |
   = note: expected type `&tera::context::Context`
              found type `tera::context::Context`
```

**Comment by sucaba** [comment]
Same

**Comment by JeanMertz** [comment]
This was changed between Tera [1.0.0-beta.13](https://docs.rs/tera/1.0.0-beta.13/tera/struct.Tera.html#method.render) (used in the latest version of Jilu) and [1.0.0-beta.17](https://docs.rs/tera/1.0.0-beta.17/tera/struct.Tera.html#method.render).

I've just released Jila 0.3.0, which includes an upgrade to Tera 1.1.0, which fixes the issue.

Apologies for the delay in tackling this.
